include/deal.II/distributed/tria.h

```diff
@@ -290,9 +290,6 @@ namespace parallel
       using active_cell_iterator =
         typename dealii::Triangulation<dim, spacedim>::active_cell_iterator;
 
-      using CellStatus =
-        typename dealii::Triangulation<dim, spacedim>::CellStatus;
```

Comment:
This is incompatible. Could this be retained?

Reply:
Done. As discussed, I have re-introduced here a C-style `enum` with the old names and same values as the new `enum class`.